api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -53,6 +52,8 @@
   private static final String WORKSPACE_ID_KEY = "WORKSPACE_ID";
   private static final String API_HOST_KEY = "API_HOST";
   private static final String BUCKET_NAME_KEY = "BUCKET_NAME";
+  private static final String CDR_VERSION_CLOUD_PROJECT = "CDR_VERSION_CLOUD_PROJECT";
```

Comment:
I think this is exactly right how you have it, but to double-check, should any of these be in a Config file?

Reply:
This was a reversion of a previous change where I'd factored these variables out for reuse.

Reply:
OK. There are also .properties files, if you need more than about this many. no worries.